internal/common/config.go

```diff
@@ -0,0 +1,17 @@
+package common
+
+import "github.com/go-openapi/swag"
+
+type Config struct {
+	EnableKubeAPI bool `envconfig:"ENABLE_KUBE_API" default:"false"`
+
+	// EnableSoftTimeouts is a boolean flag to enable Soft timeouts by assisted installer
+	EnableSoftTimeouts *bool `envconfig:"ENABLE_SOFT_TIMEOUTS"`
```

Comment:
prefer to use  a single variable that will be disabled by default, then we will not need to update the CI because no one will try to resolve installation issues in CI